static/js/dynamic-contact-form.js

```diff
@@ -200,6 +200,19 @@
       return message;
     }
 
+    // move g-recaptcha-response to marketo field
+    var marketoForm = document.querySelector('form#mktoForm');
+    marketoForm.addEventListener(
+      'submit',
+      function() {
+        var input = document.createElement('input');
+        input.type = 'hidden';
+        input.name = "grecaptcharesponse";
```

Comment:
Do you think we should try something more like "LastReCAPTCHAUserResponse" just for consistency with the other solutions out there?